tools/README.md

```diff
@@ -1,30 +1,15 @@
 openpilot tools
 ============
 
-tools to facilitate development and debugging of openpilot
 
-![Imgur](https://i.imgur.com/IdfBgwK.jpg)
-
-
-Table of Contents
+System requirements
 ============
 
-<!--ts-->
- * [Requirements](#requirements)
- * [Setup](#setup)
- * [Tool examples](#tool-examples)
-   * [Replay driving data](#replay-driving-data)
-   * [Debug car controls](#debug-car-controls)
-   * [Stream replayed CAN messages to EON](#stream-replayed-can-messages-to-eon)
- * [Welcomed contributions](#welcomed-contributions)
-<!--te-->
+openpilot tools and the following setup steps are developed and tested on:
```

Comment:
elaborate on this whole section and the setup section. this is the primary place we should link to for getting openpilot setup on your PC for development